web/src/components/tasks/business-formation/business/SuffixDropdown.tsx

```diff
@@ -22,6 +37,20 @@ export const SuffixDropdown = (): ReactElement => {
     });
   };
 
+  const getSuffixDropDownOptions = ():
+    | LlpBusinessSuffix[]
+    | LlcBusinessSuffix[]
+    | CorpBusinessSuffix[]
+    | LpBusinessSuffix[]
+    | ForeignCorpWillPracticeLawBusinessSuffix[]
+    | ForeignCorpBusinessSuffix[]
+    | NonprofitBusinessSuffix[] => {
+    if (state.formationFormData.willPracticeLaw === true) {
+      return [...foreignCorpWillPracticeLawBusinessSuffix];
+    }
+    return BusinessSuffixMap[state.formationFormData.legalType];
+  };
```

Comment:
So I thought about creating an entirely new set of LegalTypes like foreign-c-corp-will-practice-law and foreign-s-corp-will-practice-law

Rather than having the if statement above. I wasn't sure when we add legalTypes vs not and it felt like it might be overkill to do so. 

Would love more thoughts on this if other people have insight.

Reply:
@adidner: I don't think we should create additional legal types unless there are specified in our API docs. I think the way you've structured this function and are handling `willPraticeLaw` is a good and clear approach. Also, open to other folks weighing in.

Reply:
I agree with @AnJuHyppolite